README.md

```diff
@@ -51,7 +51,7 @@ digkins job build my-androip-build
   digkins login <url> [user] [password] 
 
   Stream jenkins logs for triggered build
-  digkins log <job> <queueid>       
+  digkins log <job> <buildNumber>       
```

Comment:
is  that the real build number from Jenkins? Or this other number we had before, which did not match the actual Jenkins Build NR